src/lib/containers/query_wrapper_plot_nav/QueryWrapperPlotNav.tsx

```diff
@@ -54,79 +55,85 @@ export type QueryWrapperPlotNavProps = SearchParams &
   Operator &
   OwnProps
 
-const QueryWrapperPlotNav: React.FunctionComponent<QueryWrapperPlotNavProps> = props => {
-  const [showExportMetadata, setShowExportMetadata] = React.useState(false)
-  const {
-    searchParams,
-    sql,
-    sqlOperator,
-    tableConfiguration,
-    name,
-    cardConfiguration,
-    facetsToPlot,
-    facetsToFilter,
-    hideDownload,
-    searchConfiguration,
-    limit = DEFAULT_PAGE_SIZE,
-    downloadCartPageUrl,
-  } = props
-  const sqlUsed = insertConditionsFromSearchParams(
-    sql,
-    searchParams,
-    sqlOperator,
-  )
-  
-  const entityId = parseEntityIdFromSqlStatement(sqlUsed)
-  const initQueryRequest: QueryBundleRequest = {
-    entityId,
-    concreteType: 'org.sagebionetworks.repo.model.table.QueryBundleRequest',
-    partMask:
-      SynapseConstants.BUNDLE_MASK_QUERY_COLUMN_MODELS |
-      SynapseConstants.BUNDLE_MASK_QUERY_FACETS |
-      SynapseConstants.BUNDLE_MASK_QUERY_SELECT_COLUMNS |
-      SynapseConstants.BUNDLE_MASK_QUERY_RESULTS,
-    query: {
-      sql: sqlUsed,
-      limit: limit,
-      offset: 0,
-    },
+const QueryWrapperPlotNav: React.FunctionComponent<QueryWrapperPlotNavProps> =
+  props => {
+    const [showExportMetadata, setShowExportMetadata] = React.useState(false)
+    const {
+      searchParams,
+      sql,
+      sqlOperator,
+      tableConfiguration,
+      name,
+      cardConfiguration,
+      facetsToPlot,
+      facetsToFilter,
+      hideDownload,
+      searchConfiguration,
+      limit = DEFAULT_PAGE_SIZE,
+      downloadCartPageUrl,
+    } = props
+    const sqlUsed = insertConditionsFromSearchParams(
+      sql,
+      searchParams,
+      sqlOperator,
+    )
+
+    const entityId = parseEntityIdFromSqlStatement(sqlUsed)
+    const { data: entity } = useGetEntity(entityId)
+    const initQueryRequest: QueryBundleRequest = {
+      entityId,
+      concreteType: 'org.sagebionetworks.repo.model.table.QueryBundleRequest',
+      partMask:
+        SynapseConstants.BUNDLE_MASK_QUERY_COLUMN_MODELS |
+        SynapseConstants.BUNDLE_MASK_QUERY_FACETS |
+        SynapseConstants.BUNDLE_MASK_QUERY_SELECT_COLUMNS |
+        SynapseConstants.BUNDLE_MASK_QUERY_RESULTS,
+      query: {
+        sql: sqlUsed,
+        limit: limit,
+        offset: 0,
+      },
+    }
+    return (
+      <div className="QueryWrapperPlotNav">
+        <SynapseContextConsumer>
+          {context => (
+            <QueryWrapper
+              {...props}
+              token={context?.accessToken}
+              initQueryRequest={initQueryRequest}
+            >
+              {entity && isTableEntity(entity) && entity.isSearchEnabled ? (
+                <FullTextSearch />
+              ) : (
+                <SearchV2 {...searchConfiguration} />
+              )}
```

Comment:
This is the main change.

We could just add FTS as an option in SearchV2 so users could query both across all rows and in particular columns. Was this option considered in the design phase?